web/frontend/package.json

```diff
@@ -24,9 +25,10 @@
     "dist"
   ],
   "module": "./dist/rc.js",
-  "types": "./dist/main-lib.d.ts",
+  "types": "./dist/src/main-lib.d.ts",
```

Comment:
Should we be using the [`rootDir`](https://www.typescriptlang.org/tsconfig#rootDir) option in `tsconfig.build.json` to avoid this `src`?

Reply:
I was looking for this! Thanks, fixed